libraries/botbuilder/src/fileTranscriptStore.ts

```diff
@@ -49,10 +60,10 @@ export class FileTranscriptStore implements TranscriptStore {
     }
 
     /**
-     * Get activities for a conversation (Aka the transcript)
+     * Get all activities associated with a conversation id (aka get the transcript).
      * @param channelId Channel Id.
      * @param conversationId Conversation Id.
-     * @param continuationToken Continuatuation token to page through results.
+     * @param continuationToken Continuation token to page through results.
      * @param startDate Earliest time to include.
```

Comment:
>Earliest time to include. [](start = 24, length = 25)

I've been prefixing optional params with "(Optional) " because the generated docs don't call out optional params.

Reply:
I went back and added (Optional) to all these fields.